utils/clusterTester/columnstoreClusterTester.sh

```diff
@@ -18,6 +18,9 @@ if [ -z "$COLUMNSTORE_INSTALL_DIR" ]; then
 	COLUMNSTORE_INSTALL_DIR=/usr/local/mariadb/columnstore
 fi
 
+#get temp directory
+tmpDir=`$COLUMNSTORE_INSTALL_DIR/bin/getConfig SystemConfig SystemTempFileDir`
```

Comment:
Right now there seems to be a 2nd temp dir and a lock dir that should either be tested, or what's better imo is consolidating all of those.